src/main/java/org/opensearch/jobscheduler/transport/GetJobIndexAction.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.transport;
+
+import org.opensearch.action.ActionType;
+import org.opensearch.jobscheduler.constant.CommonValue;
+
+/**
+ * Get Job Index action
+ */
+public class GetJobIndexAction extends ActionType<GetJobDetailsResponse> {
+
+    public static final String NAME = CommonValue.EXTERNAL_ACTION_PREFIX + "get/jobIndex";
```

Comment:
nit: actions are preferred to be lower case.

Reply:
can you please elaborate it with an example?

Reply:
ACK